project/project/metals.sbt

```diff
@@ -1,4 +0,0 @@
-// DO NOT EDIT! This file is auto-generated.
```

Comment:
There is another `metals.sbt` in `project/project/metals.sbt` that should be removed